src/analysis/retail/hunter/survival/modules/guide/sections/rotation/RotationSection.tsx

```diff
@@ -2,8 +2,9 @@ import { t, Trans } from '@lingui/macro';
 import { GuideProps, Section, SubSection } from 'interface/guide';
 import CombatLogParser from 'analysis/retail/hunter/survival/CombatLogParser';
 import TALENTS from 'common/TALENTS/hunter';
-import * as AplCheck from 'analysis/retail/hunter/survival/modules/apl/AplCheck';
-import { AplSectionData } from 'interface/guide/components/Apl';
+//Commented out because they will be reactived relatively quick.
```

Comment:
I know these should just be deleted but I have a bad memory and didn't want to have to remember how to re-implement them once I have the APL section working as I'd like. It's like 90% there for pack leader except for an issue with Serpent Sting debuff tracking and checking for focus levels. (Both of which I feel might be outside of my skill level to resolve or track down)